notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -7,7 +7,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.                    xtalk_frop = get_frop_wrapper(pol=pol, stream_ind=stream_ind, band_ind=spw,
</pre><p>For baselines and bands with little to no overlap in the xtalk notch and the main beam, do we want to just skip all this? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
OK, do we want to automate the check with some criterion or make a global switch?

Reply:
OK I added a global setting called CORR_MATRIX_NOTCH_CUTOFF defaulted to 30m which means by default it will skip the notch filter in the error calculation in the EW projection is longer than 30m. This skips the notch filter part by a little less than 75% of the single-baseline runs I believe.